packages/manager/apps/web-domains/src/alldoms/pages/service/serviceList/serviceList.tsx

```diff
@@ -1,21 +1,63 @@
 import React from 'react';
 import { useTranslation } from 'react-i18next';
-
 import {
   Breadcrumb,
   Datagrid,
   BaseLayout,
+  useResourcesIcebergV6,
+  ErrorBanner,
 } from '@ovh-ux/manager-react-components';
+import Loading from '@/alldoms/components/Loading/Loading';
 
 import appConfig from '@/web-domains.config';
+import { useAllDomDatagridColumns } from '@/alldoms/hooks/useAllDomDatagridColumns';
+import { useGetDatagridServiceInfoList } from '@/alldoms/hooks/data/useDatagridServiceInfoList';
+import { TServiceProperty } from '@/alldoms/types';
 
 export default function ServiceList() {
-  const { t } = useTranslation('listing');
-
+  const { t } = useTranslation('allDom');
+  const { t: tError } = useTranslation('web-domains/error');
```

Comment:
```suggestion
  const { t } = useTranslation(['allDom', 'web-domains/error']);
```
then you can use either t('xx') which will be translated to t('allDom:xx'), or t('web-domains/error:yy')